README.md

```diff
@@ -151,12 +188,13 @@ public class MainApplication extends Application implements ReactApplication {
   }
 }   
 ```
-
-
 ## Development
 
 See [development.md](./docs/development.md) for development docs.
 
+## Migration guide
+
+See [migration.md](./docs/migration.md) for migrating from older version of React Native libraries.
 
 ## Contributing
 See [CONTRIBUTING](CONTRIBUTING.md)
```

Comment:
Let's update License section too:
## Licensing
This project is licensed under the Apache V2 License. See [LICENSE](LICENSE.md) for more information.